content/en/docs/reference/kubernetes-api/ref/authentication/serviceaccount-v1.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "ServiceAccount"
+description: "ServiceAccount binds together: * a name, understood by users, and perhaps by peripheral systems, for an identity * a principal that can be authenticated and authorized * a set of secrets."
+draft: false
+collapsible: false
+weight: 1
+---
```

Comment:
If the generator code could include some simple metadata here, a Hugo layout could make use of that for sure:
```suggestion
content_type: api_reference
api_metadata:
  kind: ServiceAccount
  apiVersion: v1
  import: "k8s.io/api/core/v1"
---
```


We can [later!] localize the part of the page, too, if we want (eg “This is an API reference for `ServiceAccount/v1`; the following text is taken from the Kubernetes API definition” / vs “Ceci est la référence API de `ServiceAccount/v1`; on a copié le texte suivant du définition de l'API Kubernetes”).
Another detail, the layout could also work out `authentication` from examining the URL path / filesystem path.